Portfolio Project/index.html

```diff
@@ -9,12 +9,33 @@
 </head>
 <body>
 <header>
+    <a href="#" id="logo">My Logo is beautiful</a>
    <nav>
-        <a href="#" id="logo">My Logo is beautiful</a>
+    <ul class="navbar">
+        <li><a href="#"><i class="fa-solid fa-bars"></i></a></li>
+    </ul>
+     
    </nav>
 </header>
-<h1>Hey there,I'm Alex.</h1>
-<h2>I'm a Software Developer</h2>
-<p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+
+<main>  
+    <div id="welcome">
```

Comment:
- [ ] Kindly use `<section>` tag which contains the whole headline section. Remove this `<div>` and use `<section>` tag instead. Cheers 👍